packages/ui/package.json

```diff
@@ -11,12 +11,14 @@
         "lint": "TIMING=1 eslint --ext .tsx,.ts src/"
     },
     "devDependencies": {
-        "@babel/plugin-proposal-private-methods": "^7.18.6",
+        "@babel/plugin-proposal-decorators": "7.22.7",
```

Comment:
Plugins `@babel/plugin-proposal-decorators` and `@babel/plugin-proposal-export-default-from` were required by storybook `6.5.14`.